### DISCUSSION TITLE
Allow each note to be opened in a different Window

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#782] [Feature Requests]
Markor allows this, and that way you can edit multiple documents at the same time. It's quite awesome.

This is blocked until Flutter provides support for multiple windows.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by bergie** [comment]
This would be especially awesome with #248
